tools/compare-perf/package.json

```diff
@@ -7,7 +7,7 @@
 	"license": "GPLv2",
 	"repository": "woocommerce/woocommerce",
 	"scripts": {
-		"test": "node index.js"
+		"compare": "node index.js"
```

Comment:
Renamed this to "compare" to prevent the script from running when running.`pnpm test` at the root level.